test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -159,3 +161,25 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, cpu_only):
     if is_tf1(tensorflow_training):
         pytest.skip("This test is for TF2 only")
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_ADDONS_CMD)
+
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_CPU_INSTANCE_TYPE, indirect=True)
```

Comment:
Please add @pytest.mark.model() and @pytest.mark.model() to pass the sanity test

Reply:
Got it!